src/gatsby-theme-docz/theme/styles.js

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line
+import themeStyles from "gatsby-theme-docz/src/theme/styles";
+
+const styles = {
+  ...themeStyles,
+  inlineCode: {
+    color: "rgb(125, 137, 156)",
```

Comment:
Theme colours here?

Reply:
This has been updated to use the Jobber vars